api/src/main/java/io/strimzi/api/kafka/model/KafkaMirrorMakerSpec.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.Minimum;
+import io.sundr.builder.annotations.Buildable;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"replicas", "whitelist", "consumer", "producer", "metrics"})
+public class KafkaMirrorMakerSpec implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    public static final String DEFAULT_IMAGE =
+            System.getenv().getOrDefault("STRIMZI_DEFAULT_KAFKA_MIRRORMAKER_IMAGE", "strimzi/kafka-mirror-maker:latest");
+
+    private int replicas;
+
+    private String whitelist;
+
+    private KafkaMirrorMakerConsumerSpec consumer;
+
+    private KafkaMirrorMakerClientSpec producer;
+
+    private Map<String, Object> metrics = new HashMap<>(0);
+
+    @Description("The number of pods in the cluster.")
+    @Minimum(1)
+    @JsonProperty(required = true)
+    public int getReplicas() {
+        return replicas;
+    }
+
+    public void setReplicas(int replicas) {
+        this.replicas = replicas;
+    }
+
+    @Description("List of topics which are included for mirroring. This option allows any regular expression using Java-style regular expressions.")
```

Comment:
When you say something is a list, but it's represented as a string you need to say how the elements in the list are delimited. It's also worth giving an example in this case because you're saying it's a list of things, but also talking about regex. So is it a comma separated list of regex?